docs/guides/preferences.md

```diff
@@ -0,0 +1,326 @@
+(preferences)=
```

Comment:
should this be checked into source?

Reply:
Good question I am fine with any option. @sofroniewn @jni ?

Reply:
Is this the file that gets autogenerated? Is this the version "before autogeneration" that get's overwritten? I don't have a solid opinion, could go either way - curious what people think are pros/ cons?

Reply:
If this is generated I prefer not to commit it.

Reply:
>Is this the file that gets autogenerated? 

Yes.

Reply:
Then I am fine with @jni preference not to commit it. Don't have strong views though

Reply:
If you can drop this file @goanpeca i think this is then good to merge

Reply:
Fixed!